internal/shim-sgx/src/handler/mod.rs

```diff
@@ -71,6 +73,63 @@ impl<'a> Write for Handler<'a> {
     }
 }
 
+trait SecInfoExt {
+    fn accept(&self, dest: *const u8);
+    fn accept_copy(&self, dest: *const u8, src: *const u8);
+    fn protect(&self, dest: *const u8);
```

Comment:
do these accept any pointer or only page aligned ones?

Reply:
Shouldn't they return the error code from `rax` ??
Best to create consts for them, too.

Reply:
Oops, this is not the latest version. In the newest I uses NonNull. I'll update.